docs/docs/build/examples/wbtc-gas-fee-erc-2771/index.md

```diff
@@ -0,0 +1,84 @@
+# Gas fee payment in WBTC using ERC-2771
+
+In this example, we will show how WBTC can be used for gas fee payments using the [ERC-2771 standard](https://eips.ethereum.org/EIPS/eip-2771) on the BOB testnet. This enables users to transact without the necessity to own ether.
```

Comment:
I think we should explain a little bit about openGSN. It doesn't have to be super thorough - we can link to their docs as well. Just the basics of how it works, and how to support it in contracts. (the latter part could be under a separate header though)